packages/wordcount/src/test/index.test.js

```diff
@@ -43,10 +43,17 @@ describe( 'WordCounter', () => {
 		{
 			message: 'Punctuation.',
 			string: 'It’s two three \u2026 4?',
-			words: 3,
+			words: 4,
```

Comment:
The same sentence in Pages:
<img width="316" alt="Screen Shot 2021-05-14 at 09 31 16" src="https://user-images.githubusercontent.com/699132/118237077-27756b00-b497-11eb-9f56-0a90cc4a1d87.png">